### DISCUSSION TITLE
How do I use us-west Mining4People Server instead of na?

### DISCUSSION THREAD

**Discussion by gdi0110** [RainbowMiner/RainbowMiner#2914] [Q&A] [answered]
Mining4People has more servers for the US. How do I change the server from `na.mining4people.com` to `us-west.mining4people.com`? I have tried adding `us-west` to `Region` or `us-west.mining4people.com` to `UseHost` in the M4P section of the pool config but no luck.

**Comment by RainbowMiner** [comment]
You are right - the us-west stratum servers aren't registered in the pool module yet. I will add them asap.

**Reply by gdi0110** [reply]
Thank you so much!

**Reply by RainbowMiner** [reply, marked as answer]
The commit 9d568765da895c3360663f61ee250a223f5fc095 should fix things with the regions on Mining4People
